MapboxNavigation/Cache.swift

```diff
@@ -5,7 +5,7 @@ public typealias CompletionHandler = () -> Void
 /**
  A cache consists of both in-memory and on-disk components, both of which can be reset.
  */
-@objc(MBBimodalCache)        
+@objc(MBBimodalCache)
```

Comment:
Any idea if we need this class to be exposed to the Objective-C runtime?

Reply:
Not sure, I think it should be safe to remove it.

Reply:
Turns out `MBBimodalCache` should be exposed. Will leave it. I've also updated indentation in MapboxNavigation target to match default settings.